grid-demo/src/app/angular/lazy-loading-rows/lazy-load.service.ts

```diff
@@ -0,0 +1,55 @@
+import { ListRange } from '@angular/cdk/collections';
+import { Injectable } from '@angular/core';
+import { from, Subject } from 'rxjs';
+import { debounceTime, distinct, map, scan, mergeMap, startWith, switchMap } from 'rxjs/operators';
+import { Vm, generateVms } from 'src/app/data/vm.generator';
```

Comment:
This path should be `'./../../data/vm.generator'` (and there should be blank line).

Reply:
fixed